docs/Makefile

```diff
@@ -1,49 +1,11 @@
-# Makefile for Sphinx documentation
+# Makefile to generate additional sphinx sources
 #
 
 .DEFAULT_GOAL := help
-
-# You can set these variables from the command line.
-SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
-PAPER         =
-BUILDDIR      = _build
-
-# Internal variables.
-PAPEROPT_a4     = -D latex_paper_size=a4
-PAPEROPT_letter = -D latex_paper_size=letter
-ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
-# the i18n builder cannot share the environment and doctrees with the others
-I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
-
-.PHONY: help generate-plugins plugins/*.rst generate-stories generate-template-filters generate-autodocs clean html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest coverage gettext
+.PHONY: help generate-plugins plugins/*.rst generate-stories generate-template-filters generate-autodocs clean
 
 clean:
-	rm -rf $(BUILDDIR) stories spec code/autodocs/*.rst code/template-filters.rst
-
-##
-## Building documentation
-##
-html: generate  ## Make standalone HTML files
-	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
-	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
-
-latexpdf: generate  ## Make LaTeX files and run them through pdflatex
-	$(SPHINXBUILD) -b latex $(ALLSPHINXOPTS) $(BUILDDIR)/latex
-	@echo "Running LaTeX files through pdflatex..."
-	$(MAKE) -C $(BUILDDIR)/latex all-pdf
-	@echo "pdflatex finished; the PDF files are in $(BUILDDIR)/latex."
-
-text: generate  ## Make text files
-	$(SPHINXBUILD) -b text $(ALLSPHINXOPTS) $(BUILDDIR)/text
-	@echo
-	@echo "Build finished. The text files are in $(BUILDDIR)/text."
-
-man: generate  ## Make manual pages
-	$(SPHINXBUILD) -b man $(ALLSPHINXOPTS) $(BUILDDIR)/man
-	@echo
-	@echo "Build finished. The manual pages are in $(BUILDDIR)/man."
+	rm -rf stories spec code/autodocs/*.rst
```

Comment:
Why `code/template-filters.rst` has been excluded from the cleanup?

Reply:
Dropped by a mistake while resolving rebase conflicts?

Reply:
Could be. So, while we are at it, would it make sense to add `docs/plugins/*.rst` as well? This seem to be also omitted from the cleanup.

Reply:
Yep, makes sense.